components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2300,6 +2300,7 @@ impl<S: Snapshot> Apply<S> {
         let entries_mem_size =
             (ENTRY_MEM_SIZE * entries.capacity()) as i64 + get_entries_mem_size(&entries);
         APPLY_PENDING_BYTES_GAUGE.add(entries_mem_size);
+        APPLY_PENDING_MSGS_COUNTER.inc_by(entries.len() as i64);
```

Comment:
It should be a guage and also updated in drop method.

Reply:
Does the name `APPLY_PENDING_MSGS` suitable for that? I think it's wired.

Reply:
It can be changed to `APPLY_PENDING_ENTRIES_GAUGE`.